README.md

```diff
@@ -81,13 +82,16 @@ Maintainers are free to use stock-extracted blobs, blobs from other devices or l
 * Official builds can be released as weekly, monthly or nightlies when needed but at least they must be released on every crDroid minor version update
 * Selinux can be permissive during initial bringup days, only in beta builds. It must be enforced in every official release
 * Sepolicy must meet security standards, it must cover every system and vendor component (no random denials), however it must not contain any rules for 3rd party apps
+* Neverallows MUST NOT be masked using *SELINUX_IGNORE_NEVERALLOWS*
+* Maintainers must not disable any permissions related checks, particularly alter *ro.control_privapp_permissions*
 * Official builds must not be released with gapps unless it is impossible to flash them without bootloop
-* If released with Gapps, you mush include our gapps version found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
+* If releasing with Gapps, you must include our gapps variant found at our [GitLab](https://gitlab.com/crdroidandroid/android_vendor_gapps)
```

Comment:
suggest to drop this
we can't validate and does less work for us to maintain gapps

Reply:
As long as @neobuddy89 wants to maintain our gapps I would keep it.

Reply:
we done even have a 12.1 branch
as I said, less work, better for us
https://gitlab.com/crdroidandroid/android_vendor_gapps

Reply:
Im waiting for Pranav's answer, if he doesnt want to maintain it I will change this rule to enforce the most basic variant of gapps.

Reply:
Removed our gapps reference.